### PR TITLE
Writing posts

### PR DESCRIPTION
**Pull request by Pe-De-E**
Added a form to the admin-backend, so that you can write posts there.

### CODE REVIEW

**Review by JessicaLoers — COMMENTED** [review]
**Hi @PhilippDemmelmair** ✌️
I have some suggestions regarding your from "validation" .

And please check the following file `api/teaser/[articleId].js`
I would recommend you use `try/catch` 🤔
```
GitHub Actions
/ Code Quality
Expected a default case
```

File: src/pages/admin.js [hunk 1]
```diff
@@ -44,3 +90,50 @@ const Comments = styled.article`
...
 	color: white;
 `;
+
+const BlogPostEntry = styled.form``;
+
+const TitleLabel = styled.label`
+	display: block;
+	width: 40%;
+	margin: 2% auto;
+	padding: 1%;
```
[the remark this answers is not in the document]
Comment (PhilippDemmelmair) [reply src/pages/admin.js:100]: But it looks very neat in the mobile view. Trust me. All the cool kids use 1%....

File: src/pages/admin.js [hunk 2]
```diff
@@ -14,7 +29,40 @@ export default function AdminPage() {
...
-				New Post<br></br>Here will be a form to create new posts;
+				<BlogPostEntry
+					onSubmit={event => {
+						event.preventDefault();
+						if (post.title.length >= 10 && post.text.length >= 100) {
+							addPost(post);
+							setPost({...post, title: '', text: ''});
+						} else {
+							alert(
+								'Der Titiel muss mindestens zehn Zeichen und der Text mindestens 100 Zeichen lang sein.'
```
Comment (JessicaLoers) [inline src/pages/admin.js:40]: There's still a litte typo `Der Titel `

**Review by JessicaLoers — APPROVED** [review]

File: src/pages/admin.js (hunk 2 above)
Comment (PhilippDemmelmair) [reply src/pages/admin.js:40]: damn,.... war ein test,...

Comment (JessicaLoers) [reply src/pages/admin.js]: for sure 😁

**Review by JessicaLoers — APPROVED** [review]

**Pull request merged**